src/main/java/frc/robot/commands/auto/AutoHubShoot.java

```diff
@@ -0,0 +1,25 @@
+/* ==================================================
+ * Authors: Lucas Jacobs
+ * --------------------------------------------------
+ * Description:
+ * This command group shoots the ball into the top hub 
+ * assuming the front of the robot is pressed agains the hub.
+ * 
+ * The distance from the fender to the centre of the hub is 86 cm
+ * (From Game Manual)
+ * ================================================== */
+
+package frc.robot.commands.auto;
+
+import edu.wpi.first.wpilibj2.command.ParallelCommandGroup;
+import edu.wpi.first.wpilibj2.command.SequentialCommandGroup;
+import frc.robot.Robot;
+
+public class AutoHubShoot extends SequentialCommandGroup {
+    public AutoHubShoot(Robot robot) {
+        super(
+            new AutoDrive(robot, -.8, 0, .6), // drive backwards
+            new AutoShoot(robot, .38, .8) //shoot
```

Comment:
In theory, if the `timeToShot` is greater than the drive back time, they could be run in parallel instead of sequential so that the shooter charges up as the robot drives back, but that only saves 0.6 seconds lol so don't feel the need to change it if it works well.

Reply:
That's what I did originally, but I was having problems with 2 Timer objects running at the same time. However, that was before I added `timer.reset()` everywhere, so that may have fixed the problem. I can't test it without the robot though, so I'll do it after the break.